### PR TITLE
Translate UCR Report Columns

### PR DESCRIPTION
**Pull request by NoahCarnahan**
This PR Allows users to specify multiple languages for report column headings. Django's `request.LANGUAGE_CODE` is used to determine which language is displayed to the user viewing the reports. Example column spec:

``` json
{
    "type": "field",
    "field": "owner_id",
    "display": {
        "en": "Name",
        "he": "שם"
    },
    ...
}
```

Strings can still be used in the spec if the report builder doesn't want to specify any translations:

``` json
{
    "type": "field",
    "field": "owner_id",
    "display": "Name",
    ...
}
```

Related features that we will probably want eventually:
- Translations of filter display values
- Set language for emailed reports (currently will send in the language of the user who scheduled the report)

### CODE REVIEW

**Comment by NoahCarnahan** [comment]
I'm having trouble getting `request.LANGUAGE_CODE` to change in my local environment, so hold off on merging until I confirm that that works.

**Comment by NoahCarnahan** [comment]
cc @nickpell

**Comment by NoahCarnahan** [comment]
(also test failure is real)

File: corehq/apps/app_manager/app_strings.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def maybe_add_index(text):
                 yield id_strings.report_name_header(), 'Report Name'
                 yield id_strings.report_description_header(), 'Report Description'
                 for column in config.report.report_columns:
-                    yield id_strings.report_column_header(config.report_id, column.column_id), column.display
+                    yield id_strings.report_column_header(config.report_id, column.column_id), column.get_header(lang)
```
Comment (dimagimon) [inline corehq/apps/app_manager/app_strings.py]: E501 line too long (118 > 115 characters)

**Comment by NoahCarnahan** [comment]
I replaced `request.LANGUAGE_CODE` with `request.couch_user.language`. `LANGUAGE_CODE` could only ever be any of the languages in the `LANGUAGES` setting, which was much smaller than the list of possible languages for an HQ user. Should be good to merge now if tests pass.

**Comment by NoahCarnahan** [comment]
never mind, think I found an issue...

**Comment by czue** [comment]
this looks great. possible to add a test?

File: corehq/apps/userreports/tests/test_report_config.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from corehq.apps.userreports.exceptions import BadSpecError
 from corehq.apps.userreports.models import ReportConfiguration, DataSourceConfiguration
 from corehq.apps.userreports.reports.factory import ReportFactory
+from corehq.apps.userreports.reports.specs import FieldColumn
```
Comment (dimagimon) [inline corehq/apps/userreports/tests/test_report_config.py]: F401 'FieldColumn' imported but unused

**Comment by NoahCarnahan** [comment]
@czue tests added

**Comment by czue** [comment]
thanks for merge and sorry, this fell off my radar

**Comment by NoahCarnahan** [comment]
No worries!

**Pull request merged**